lang/py/avro/schema.py

```diff
@@ -587,6 +587,15 @@ def symbols(self) -> Sequence[str]:
             return symbols
         raise Exception
 
+    @property
+    def default(self) -> Union[str, None]:
+        symbol = self.get_prop("default")
+        if isinstance(symbol, str):
+            return symbol
+        if symbol is None:
+            return None
+        raise Exception
```

Comment:
Please consider raising a more specific exception in this case. Would [avro.errors.InvalidDefault](https://github.com/apache/avro/blob/895c3db01860c44eef9f21de642b63049550366b/lang/py/avro/errors.py#L47) make sense?